Project1/styles.css

```diff
@@ -0,0 +1,51 @@
+img {
+    max-width: 100%;
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+figcaption {
+    text-indent: 5%;
+    font-size: 13px;
+    font-family: helvetica, sans-serif;
+}
+
+h1 {
+    color: black;
+    text-align: center;
+    font-family: helvetica, sans-serif;
+    font-size: 45px;
+}
+
+h2 {
+    text-align: center;
+    font-size: 20px;
+    font-style: normal;
```

Comment:
You can safely delete this line; the default `font-style` for headings is already `normal`.
```suggestion
```